test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -4138,6 +4355,249 @@ TEST_P(SslSocketTest, TicketSessionResumptionDifferentServerCertDifferentSAN) {
                               version_);
 }
 
+TEST_P(SslSocketTest, TicketSessionResumptionAcceptUntrustedNoClientCertificate) {
```

Comment:
It would be better for all these resumption tests to make a new parameterized test `SslSocketResumptionTest` and have parameters for tls1.2 server session, tls1.2 ticket, and tls1.3. Is that feasible, or are there too many differences to make that work?